src/routes/pages/SignUp.tsx

```diff
@@ -0,0 +1,5 @@
+const SignUp = () => {
```

Comment:
Page componentでも戻り値の型を書くようにしましょう。React.FCで十分かなと思います。(React 17だとpropsにchildrenが含まれているけど)

```suggestion
const SignUp: React.FC = () => {
```

Reply:
「(React 17だとpropsにchildrenが含まれているけど)」
今までこの理由でReact.FCを使う習慣がありませんでした！
しかし、形を書いた方が良いですね！ありがとうございます。